app/repository/family.py

```diff
@@ -44,13 +44,13 @@ def _get_query(db: Session) -> Query:
         db.query(Family, Geography, FamilyMetadata, Organisation)
         .join(Geography, Family.geography_id == Geography.id)
         .join(FamilyMetadata, FamilyMetadata.family_import_id == Family.import_id)
-        .join(
-            FamilyOrganisation, FamilyOrganisation.family_import_id == Family.import_id
-        )
-        .join(Organisation, FamilyOrganisation.organisation_id == Organisation.id)
+        .join(FamilyCorpus, FamilyCorpus.family_import_id == Family.import_id)
+        .join(Corpus, Corpus.import_id == FamilyCorpus.corpus_import_id)
+        .join(Organisation, Corpus.organisation_id == Organisation.id)
     )
 
 
+# TODO: Remove use of FamilyOrganisation
```

Comment:
Pernickety comment - but should this say remove use of `MetadataOrganisation`?

Reply:
haha - yes both!